components/ui/ProjectPublishingChecklist.vue

```diff
@@ -290,17 +308,27 @@ export default {
           },
         },
         {
-          condition: this.project.status === 'draft',
-          title: 'Submit for review',
+          condition: true, // This condition is handled for us in the root div
+          title: `${
```

Comment:
Think this may have been a bit more readable if it was implemented as two separate nags with `hide` conditions